<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>530</y>
      <width>94</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>511</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>General settings</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>821</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Basic info</string>
      </property>
      <widget class="QPushButton" name="adjPlot">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>70</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
      <widget class="QLabel" name="spatialLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Spatial info file</string>
       </property>
      </widget>
      <widget class="QLabel" name="skullLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Skull file</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="skullFilename">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>471</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="skullLoad">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>100</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="spatialFilename">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>471</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="adjFilename">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>471</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="adjFnameButton">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>70</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
      <widget class="QPushButton" name="brainLoad">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>70</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QLabel" name="adjLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Adjacency file</string>
       </property>
      </widget>
      <widget class="QPushButton" name="spatialFnameButton">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>40</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
      <widget class="QPushButton" name="skullFnameButton">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>100</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
      <widget class="QPushButton" name="skullPlot">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>100</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="thresholdValue">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>40</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="thresholdLabel">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
      <zorder>adjPlot</zorder>
      <zorder>spatialLabel</zorder>
      <zorder>skullLabel</zorder>
      <zorder>skullFilename</zorder>
      <zorder>skullLoad</zorder>
      <zorder>spatialFilename</zorder>
      <zorder>adjFilename</zorder>
      <zorder>adjFnameButton</zorder>
      <zorder>brainLoad</zorder>
      <zorder>adjLabel</zorder>
      <zorder>spatialFnameButton</zorder>
      <zorder>skullFnameButton</zorder>
      <zorder>skullPlot</zorder>
      <zorder>thresholdValue</zorder>
      <zorder>thresholdLabel</zorder>
      <zorder>subPlotBox</zorder>
     </widget>
     <widget class="QGroupBox" name="subPlotBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>791</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Add properties and sub-plots</string>
      </property>
      <widget class="QPushButton" name="subPlotButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>60</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="subPlotValue">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>60</y>
         <width>113</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="subPlotPropLabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Property</string>
       </property>
      </widget>
      <widget class="QLabel" name="subPlotValueLabel">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="subPlotName">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>60</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="subPlotNameLabel">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>40</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot name (can be left blank)</string>
       </property>
      </widget>
      <widget class="QLabel" name="subPlotPropLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Brain</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="subPlotValue_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>90</y>
         <width>113</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="propsFilename">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>501</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="propsLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Properties file</string>
       </property>
      </widget>
      <widget class="QPushButton" name="propsFnameButton">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>120</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
      <widget class="QComboBox" name="subPlotBrain">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>191</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="propsLoad">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>120</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="subPlotProp">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>113</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>521</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Figure options</string>
      </property>
      <widget class="QPushButton" name="figureSaveButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>94</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Save as...</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="showFigCheckBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>show figure</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="clearFigButton">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>94</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear figure</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Adjust plot properties</string>
     </attribute>
     <widget class="QTreeWidget" name="plotTree">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>461</width>
        <height>421</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Plot type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Plot name</string>
       </property>
      </column>
     </widget>
     <widget class="QSlider" name="opacitySlider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>160</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="opacityLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Opacity</string>
      </property>
     </widget>
     <widget class="QSlider" name="redSlider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="greenSlider">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="blueSlider">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="colourLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Colour</string>
      </property>
     </widget>
     <widget class="QLabel" name="redLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
     <widget class="QLabel" name="greenLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
     <widget class="QLabel" name="blueLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="redValueBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="greenValueBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="blueValueBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>210</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="visibleCheckBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot visible</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="opacityBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Analysis</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="log">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>801</width>
        <height>411</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:9pt;&quot;&gt;# Python code to repeat what you've just done with the buttons&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>94</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>save to file</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_fig"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_fig">
   <property name="text">
    <string>Save fig</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
